### ISSUE TITLE
Support Node 13 - increase ext2fs version in shrinkwrap

### ISSUE THREAD

**Issue by renesy** [balena-io/balena-cli#1591]
- **balena CLI version:** 11.25.1
- **Operating system version:** Arch Linux Kernel 5.4.15
- **32/64 bit OS and processor:** 64-bit Intel
- **Install method:** npm
- **If npm install, Node.js and npm version:** Node 13.7.0, npm 6.13.6

---

The [npm-shrinkwrap.json](https://github.com/balena-io/balena-cli/blob/master/npm-shrinkwrap.json#L5154) sets ext2fs to version 1.0.29, which prevents building on Node 13. Please consider using the latest ext2fs v1.0.31 which builds fine on Node 13.

**Comment by pdcastro** [comment]
The `ext2fs` dependency appears to come from the [resin-image-fs](https://github.com/balena-io-modules/resin-image-fs) package:
```
$ npm list
...
├─┬ resin-image-fs@5.0.8
│ ├─┬ ext2fs@1.0.29
```
I've created issue balena-io-modules/resin-image-fs/issues/47 in the upstream repo.

**Comment by renesy** [comment]
Yes, the dependency is indeed from resin-image-fs. However, it is this packages npm-shrinkwrap.json which fixes the version to 1.0.29.
resin-image-fs itself doesn't fix the version and upstream would use the latest 1.0.31.

**Comment by pdcastro** [comment]
`ext2fs` was updated as requested in CLI version 11.25.10. Thank you for reporting it! 👍
